cli/internal/helm/loader.go

```diff
@@ -157,9 +169,11 @@ func (i *ChartLoader) loadRelease(info chartInfo, helmWaitMode helm.WaitMode) (h
 		updateVersions(chart, compatibility.EnsurePrefixV(constants.VersionInfo()))
 
 		values, err = i.loadConstellationServicesValues()
+	case awsLBControllerInfo.releaseName:
+		values, err = i.loadAWSLoadBalancerControllerValues()
 	}
 
-	if err != nil {
+	if err != nil || values == nil {
```

Comment:
I don't think it's good practice to check the return value if there is also an error returned. All functions above should return an error if they don't return the values. So essentially we check if any of the functions was called. Therefore, I think that a default statement which returns an error is clearer.